generators/options.js

```diff
@@ -33,10 +44,16 @@ const commonOptions = {
     desc: 'Compose with a generator at current project',
     type: Boolean,
   },
+  regenerate: {
+    desc: 'Regenerate a generator at current project',
```

Comment:
Regenerate all sequence of generators?

Reply:
> Regenerate the entire generators sequence?

This is the default for now.
`--regenerate` is to regenerate only the current generator.

Reply:
And... it works? :-O. I'm not able to imagine that it could work without any side effect on the other generated code

Reply:
It would require https://github.com/jhipster/generator-jhipster/issues/15621.
There are 2 alternatives:
- Implement https://github.com/jhipster/generator-jhipster/issues/15621
- Drop add/regenerate support.